src/bridge/ERC20Bridge.sol

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity ^0.8.4;
+
+import "./AbsBridge.sol";
+import "./IERC20Bridge.sol";
+import "../libraries/AddressAliasHelper.sol";
+import {InvalidTokenSet, CallTargetNotAllowed} from "../libraries/Error.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+/**
+ * @title Staging ground for incoming and outgoing messages
+ * @notice Unlike the standard Eth bridge, native token bridge escrows the custom ERC20 token which is
+ * used as native currency on L2.
+ */
+contract ERC20Bridge is AbsBridge, IERC20Bridge {
+    using SafeERC20 for IERC20;
+
+    /// @inheritdoc IERC20Bridge
+    address public nativeToken;
```

Comment:
We should add a storage gap field to the end of AbsBridge similar to the openzeppelin upgradeable contracts, so we can add new fields to AbsBridge without them conflicting with this field of ERC20Bridge

Reply:
done